apps/web/.env.test

```diff
@@ -31,18 +31,12 @@ API_HOST=http://test
 # Debug module, examples
 # DEBUG=express-session
 
-AUTH_CLIENT_ID=<value goes here>
-AUTH_CLOUD_INSTANCE_ID=<value goes here>
-AUTH_TENANT_ID=<value goes here>
-AUTH_CLIENT_SECRET=<value goes here>
-AUTH_REDIRECT_URI=<value goes here>
+AUTH_CLIENT_ID=auth_client_id
+AUTH_CLOUD_INSTANCE_ID=auth_cloud_instance_id
+AUTH_TENANT_ID=auth_tenant_id
+AUTH_CLIENT_SECRET=auth_client_secret
+AUTH_REDIRECT_URI=auth_redirect_uri
```

Comment:
This should be `/auth/redirect` even in test as that is how the Azure app has been configured and the route on our side is like that.

Reply:
Ok... but should this even be an env variable then if it's fixed? I see there is `redirectUri: process.env.AUTH_REDIRECT_URI || '/auth/redirect',` in the config setup, which suggests it can be overriden?

Reply:
During testing or other env changes it will come handy to have it as an ENV as we can change it without building and deploying the entire solution again. Its one of those that stay like that long term but should have the option to change it on the fly if need be and skip a build step.

Reply:
If we change the .env.test to use `/auth/redirect` then that wouldn't prove that the environment variable is being picked up, as it'd be identical to the default value. If we have the option to change it, then the test needs to be able to identify between two possible values...